_blinka/pine64.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: download
+board_id: "pine64"
+title: "Pine64 Download"
+name: "Pine64"
+manufacturer: "Raspberry Pi Foundation"
+board_url: "https://store.pine64.org/?product=pine-a64-board"
+board_image: "pine64.png"
+download_instructions: "https://learn.adafruit.com/circuitpython-on-orangepi-linux/circuitpython-orangepi"
+downloads_display: false
+blinka: true
+features:
+  - Wi-Fi
+  - Bluetooth/BLE
+  - Ethernet
+  - HDMI
+  - 40-pin GPIO
+---
+
+The Pine64 is the first pine64.org Single Board Computer powered by Allwinner’s A64 Quad-Core ARM Cortex A53 64-Bit SOC
+
+- Allwinner A64 Quad Core SOC with Mali 400 MP2 GPU
+- DDR3 RAM (up to 2GB)
+- Gigabit Ethernet
+- Micro SD
+- HD Digital Video Out
+- 2x USB 2.0 Host
+- Euler Bus
+- Expansion Bus
+- PI-2 Bus
+- Touch Panel Port (TP)
+- Display Serial Interface (DSI) Port
+- IR R/X Port
+- Real Time Clock Port (RTC)
+- Lithium Battery Port
+- CMOS Sensor Interface Port
+- Optional 802.11GBN + BT 4.0 module expansion module
+- Headphone Jack
+- Micro USB Power (5V 2A) Port
+
+## Purchase
+* [pine64 2GB](https://store.pine64.org/?product=pine-a64-board-2gb)
+* [pine64 1GB](https://store.pine64.org/?product=pine-a64-board-1gb)
+* [pine64 521MB](https://store.pine64.org/?product=pine-a64-board)
```

Comment:
This should be 512MB

Reply:
Hey! 

The link is right https://store.pine64.org/?product=pine-a64-board points directly to the 512MB one.

Reply:
Yes, but the text says 521MB, which is about 9MB more than it should be ;)

Reply:
hahahahaha my bad.